pool/app/message_template/default/locales/default_de.ml

```diff
@@ -547,3 +547,44 @@ let contact_registration_attempt =
   ; sms_text
   }
 ;;
+
+let user_import =
+  let label = Label.UserImport in
+  let email_text =
+    [ p
+        [ txt "Ihr Account wurde kürzlich importiert."
```

Comment:
We should crosscheck the message with the recruitment team. May we should use migration instead of import.

Reply:
migration sounds better. But yes, we should ask the recruitment team to have a look at all our message templates. They can be viewed in the UI as well.